frontend/src/components/managePermissions/UserDataTable.vue

```diff
@@ -232,8 +232,15 @@ function deleteAssignment(assignment: FamApplicationUserRoleAssignmentGet) {
     border: none;
 }
 
+.data-table-container {
+    margin: 0.05rem;
+    background: $light-layer-one;
+    z-index: -1;
+    min-height: calc(100vh - 18.745rem);
```

Comment:
Hi Jota, if you check our main branch, we don't set background color here anymore. The background color is set in the `frontend/src/components/managePermissions/ManagePermissions.vue` file. 
<img width="431" alt="Screen Shot 2023-10-10 at 1 14 38 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/0e47872a-ec2a-4d77-a4b3-0301a88b81a7">

Your changes break the style for notification message, currently on your branch, the notification message margin is not correct.
![image](https://github.com/bcgov/nr-forests-access-management/assets/23131735/94b98d4c-e5b5-46ba-9804-e35182120544)

But if you check the FAM-dev, it looks good:
![image](https://github.com/bcgov/nr-forests-access-management/assets/23131735/ab649ad4-c88b-4588-8eaa-fe12fd0e3229)

Please check if you really need this `data-table-container` class anymore.

Reply:
Sure, I'll be looking into that to confirm if it's necessary. Thanks!